action.yml

```diff
@@ -19,8 +19,15 @@ inputs:
   ecr_repository:
     description: "Ecr repository name"
     required: false
+  sonar_project_key:
+    description: "When using a monorepo, you may need to specify the project key"
+    required: false
+  sonar_project_base_dir:
+    description: "When using a monorepo, you may need to specify the base directory for your project"
+    required: false
+    default: './'
 runs:
   using: "docker"
-  image: docker://public.ecr.aws/p4v7w0a5/lazy/nodejs-action:v1.1.5
```

Comment:
After PR gets merged, image will be: `docker://public.ecr.aws/p4v7w0a5/lazy/nodejs-action:v1.1.6`

Reply:
Is it problematic to just leave it as "Dockerfile"? I didn't notice much of a difference in deploy speed.

Reply:
Also, docker recommends doing it this way, which is why I changed it

Reply:
This will build the image every time locally instead of pulling from the ECR

Reply:
yea but github will cache the image after it is built once, so it doesn't impact performance that much

Reply:
Please don't use Dockerfile there. GitHub doesn't cache. We use self hosted runners where we clear the cache every few hours. Building everytime will be prone to issues because of the changes to the base image and other packages which is out of our control.

Reply:
ok i will update then